src/riverroute/RtmSpmd.F90

```diff
@@ -50,7 +37,7 @@ subroutine RtmSpmdInit(mpicom)
 
       ! Get my processor id
       call mpi_comm_rank(mpicom_rof, iam, ier)
-      if (iam == MAINTASK) then
+      if (iam == 0) then
```

Comment:
Why change this line?  I think that the constant MAINTASK is better.

Reply:
Okay - I'll move it back. My reasoning was fewer variable declarations.